x-pack/plugins/security_solution/common/endpoint/data_generators/sentinelone_data_generator.ts

```diff
@@ -26,6 +29,21 @@ import type {
 } from '../types';
 
 export class SentinelOneDataGenerator extends EndpointActionGenerator {
+  static readonly scriptExecutionStatusValues: Readonly<
+    Array<SentinelOneRemoteScriptExecutionStatus['status']>
+  > = Object.freeze([
+    'canceled',
+    'completed',
+    'created',
+    'expired',
+    'failed',
+    'in_progress',
+    'partially_completed',
+    'pending',
+    'pending_user_action',
+    'scheduled',
```

Comment:
These are used throughout this PR. Perhaps we could move them to an enum? I always nitpick these enums because they make it easier for the next person working with this code to find usages simply by clicking the name :)

Reply:
Np. Always provide the feedback. 

Personally, I don't like ENUMs as they are a bit weird to work with - it just the way TS has implemented them. It also (in allot of cases) forces us to cast simple string values to the ENUM values when trying to use them. 

if you are ok, I'll leave it as is.